### ISSUE TITLE
Ratio in menu for code/pdf windows dividing

### ISSUE THREAD

**Issue by sergiokapone** [texstudio-org/texstudio#449]
It is possible to provide a feature, for dividing code window and pdf window in some proportion/ Example from Total Commander

[picture](https://i107.fastpic.ru/big/2018/1227/ea/f2e6f9788ae6ae789b02f40b0c652aea.png)

**Comment by timhoffm** [comment]
What is the purpose of such a feature? You can modify the ratio be dragging the splitter line between those two. IMHO that's better, because you can adapt the size with immediate visual feedback. You won't get exact ratios, but there's benefit in them. Also, I wouldn't know which abtract x/y ratio I would need for a good result.

**Comment by sergiokapone** [comment]
Yes, it may be good for a perfectionists, sometimes it is necessary to quickly move from the ratio of 70/30 to 30/70 without dragging the separator. In any case, I do not insist.
